src/dmd/lexer.d

```diff
@@ -146,6 +146,31 @@ class Lexer
         }
     }
 
+    /**************************************
+     * Reset lexer to lex #define's
+     */
+    final void resetDefineLines(const(char)[] slice)
```

Comment:
Move to cparse, its only user.

Reply:
It accesses private Lexer data.

Reply:
Then make it protected? :-)

Reply:
I never much cared for protected. Having a function to do it is better encapsulation.